warehouse/forklift/legacy.py

```diff
@@ -143,8 +143,8 @@ def namespace_stdlib_list(module_list):
     "11",
 }
 
-# manylinux pep600 is a little more complicated:
-_manylinux_platform_re = re.compile(r"manylinux_(\d+)_(\d+)_(?P<arch>.*)")
+# manylinux pep600 and musllinux pep656 are a little more complicated:
+_manylinux_re = re.compile(r"(?P<libc>m(any|usl))linux_(\d+)_(\d+)_(?P<arch>.*)")
```

Comment:
I'm wondering if there would be the slightest impact in performance bewteen `m(any|usl)` versus the more readable `(many|musl)`."Readability counts".

Reply:
Ah yeah, no reason not to swap it